simulation-scripts/perturb.sh

```diff
@@ -292,60 +250,6 @@ run_scripts() {
 
 }
 
-run_cleanup() {
```

Comment:
Can you remind me why we no longer cover tracks? This is also validating the scenario, is that behaviour somewhere else now?

Reply:
Is not run by simulator as its assumed that the infra will be recreated for each scenario at the moment